pvlib/inverter.py

```diff
@@ -114,9 +114,9 @@ def sandia(v_dc, p_dc, inverter):
     References
     ----------
     .. [1] D. King, S. Gonzalez, G. Galbraith, W. Boyson, "Performance Model
-       for Grid-Connected Photovoltaic Inverters", SAND2007-5036, Sandia
-       National Laboratories.
-
+       for Grid-Connected Photovoltaic Inverters", Sandia National
+       Laboratories, Albuquerque, N.M., USA, SAND2007-5036, Sept. 2007.
+       :doi:`10.2172/920449`
     .. [2] System Advisor Model web page. https://sam.nrel.gov.
```

Comment:
Perhaps we could improve this reference as well. Looking at the [citing SAM webpage](https://sam.nrel.gov/citing-sam.html), I can't see a specific reference for the database. If we just want a general reference, as the current reference is supposed to be, we could go for something like:
```suggestion
    .. [2] System Advisor Model™ Version 2025.4.16 (SAM™ 2025.4.16). National Renewable Energy Laboratory. Golden,
            CO. Accessed August 13, 2025. https://https://sam.nrel.gov .
```

This is copied from the [citing SAM webpage](https://sam.nrel.gov/citing-sam.html)

Reply:
I don't think we should cite a particular verion of SAM, since pvlib doesn't promise that it distributes files from that version. Replacing the CEC module and inverter files with more up-to-date versions is #1345